liquid.py

```diff
@@ -254,20 +267,42 @@ def set_volumes_paths(substitutions={}):
     substitutions['liquid_volumes'] = config.liquid_volumes
     substitutions['liquid_collections'] = config.liquid_collections
 
-    pwd = os.path.realpath(os.path.dirname(__file__))
-    repos = [
-        ('hoover', 'snoop2', '/opt/hoover/snoop'),
-        ('hoover', 'search', '/opt/hoover/search'),
-        ('liquidinvestigations', 'core', '/app'),
-    ]
-    for (org, repo, target_path) in repos:
-        repo_path = os.path.join(pwd, 'repos', org, repo)
-        repo_volume_line = (f'"{repo_path}:{target_path}",\n')
-        key = f'{org}_{repo}_repo'
+    repos_path = os.path.join(os.path.realpath(os.path.dirname(__file__)), 'repos')
+    repos = {
```

Comment:
You can simplify this by extracting some variables:

```python3
li_path = config.mount_li_path or os.path.join(repos_path, 'liquidinvestigations')
hoover_path =  ...

repos = {
    ....
        'local': li_path,
    ....
}
```

And then remove the following two `if`s.

Reply:
I need `abspath(config.mount_li_path)` (from nomad docs: "Use absolute paths to mount arbitrary paths on the host"), so I can't make this simplification: `abspath(None)` raises exception and treating it would be actually more "expensive".